aspnetcore/signalr/get-started-signalr-core.md

```diff
@@ -78,24 +120,66 @@ The SignalR server must be configured so that it knows to pass requests to Signa
 
   [!code-csharp[Startup](get-started-signalr-core/sample/Startup.cs?highlight=22,40-43)]
 
+# [Visual Studio Code](#tab/visual-studio-code)
+
+1. To configure a SignalR project, modify the `ConfigureServices` method of the application's `Startup` class by inserting a call to `services.AddSignalR`.
```

Comment:
~~modify the `ConfigureServices` method of the application's `Startup` class~~
modify the project's `Startup.ConfigureServices` method